### PR TITLE
Add all Hybrids of AnythingsMC

### PR DESCRIPTION
**Pull request by TonimatasDEV**
Add all Hybrids softwares of [AnythingsMC Hybrids](https://anythingmc.xyz/server/info/lists/server_software/#plugins-and-mods-hybrids):
- Contigo
- NeptuneForge
- PFServer
- Lava
- AtomMC
- Hotpur
- Kettle

### CODE REVIEW

File: java/MODS+PLUGINS.md [hunk 1]
```diff
@@ -87,6 +87,73 @@ This list contains Minecraft Java mods/plugins server softwares.
 - **Plugins:** Bukkit, Spigot
 - **Mods:** Forge
 
+### [♓ Contigo](https://github.com/CyberdyneCC/Thermos)
```
Comment (UeberallGebannt) [inline java/MODS+PLUGINS.md:90]: This link leads to thermos, not Contigo

File: java/MODS+PLUGINS.md [hunk 2]
```diff
@@ -87,6 +87,73 @@ This list contains Minecraft Java mods/plugins server softwares.
...
+- **Plugins:** Bukkit, Spigot
+- **Mods:** Forge
+
+### [♓ NeptuneForge](https://github.com/NeptunePowered/NeptuneForge)
+- **Alternative**: -
+- **Version:** -
+- **Author:** NeptunePowered
+- **Fork:** Forge/Bukkit --> NeptuneForge
+- **Description:** A CanaryLib implementation for Minecraft Forge.
+- **Plugins:** Bukkit, Spigot
```
Comment (UeberallGebannt) [inline java/MODS+PLUGINS.md:105]: NeptuneForge is not implementing Bukkit or Spigot, it's implementing CanaryLib which is a fork of hMod. Therefore the Fork chain should be hMod --> CanaryMod + Forge --> NeptuneForge.

File: java/MODS+PLUGINS.md [hunk 3]
```diff
@@ -87,6 +87,73 @@ This list contains Minecraft Java mods/plugins server softwares.
...
+- **Alternative**: CatServer
+- **Version:** 1.12.2
+- **Author:** Timardo
+- **Fork:** Forge/Spigot --> Lava
+- **Description:** Lava is a 1.12.2 Forge/Spigot hybrid server not forked from any other hybrid servers.
+- **Plugins:** Bukkit, Spigot
+- **Mods:** Forge
+
+### [🔆 AtomMC](https://github.com/josephworks/AtomMC)
+- **Alternative**: -
```
Comment (UeberallGebannt) [inline java/MODS+PLUGINS.md:127]: Maybe add an alternative like Magma

File: java/MODS+PLUGINS.md [hunk 4]
```diff
@@ -87,6 +87,73 @@ This list contains Minecraft Java mods/plugins server softwares.
...
+- **Version:** 1.12.2
+- **Author:** josephworks
+- **Fork:** Forge/Spigot/Craftbukkit -->  AtomMC
+- **Description:** Atom is a Minecraft Server Software that is based on Minecraft Forge and CraftBukkit for 1.12.2 version of Minecraft.
+- **Plugins:** Bukkit, Spigot, Paper
```
Comment (UeberallGebannt) [inline java/MODS+PLUGINS.md:132]: As there is no Paper code in the AtomMC repository, it shouldn't support Paper.

File: java/MODS+PLUGINS.md [hunk 5]
```diff
@@ -87,6 +87,73 @@ This list contains Minecraft Java mods/plugins server softwares.
...
+- **Version:** 1.12.2, 1.14.4
+- **Author:** KettleFoundation
+- **Fork:** Forge/Bukkit --> MCPC+ --> Cauldron --> kCauldron --> Thermos --> Contigo --> Kettle
+- **Description:** A fork of Purpur that aims to improve performance and add FabricMC compatibility.
+- **Plugins:** Bukkit, Spigot
+- **Mods:** Forge
+
+TableCloth - A Hybrid jar forked from AtomMC combining Forge + Spigot and a couple of Paper Patches.
+Version: Unknown
+Discontinued
```
Comment (UeberallGebannt) [inline java/MODS+PLUGINS.md:155]: Format is missing here

**Review by UeberallGebannt — CHANGES_REQUESTED** [review]
Please take a look at my requests, other than that it looks good to me :)

**Comment by TonimatasDEV** [comment]
Everything fixed except TableCloth, I couldn't find information about it

**Comment by TonimatasDEV** [comment]
> Please take a look at my requests, other than that it looks good to me :)

Everything fixed except TableCloth, I couldn't find information about it

**Review by UeberallGebannt — APPROVED** [review]

**Pull request merged**